lxd/instance/drivers/driver_qemu.go

```diff
@@ -5157,3 +5157,40 @@ func (vm *qemu) writeInstanceData() error {
 
 	return nil
 }
+
+// Info returns "qemu" and the currently loaded qemu version, 0 if not found.
+func (vm *qemu) Info() instance.Info {
+	var qemuVersion, qemuBinary, qemuPath string
+	var out []byte
+	var err error
+	data := instance.Info{
+		Name:    "qemu",
+		Version: "0",
```

Comment:
I think this should default to being empty, so if qemu isn't found on the system, we don't erroneously report it in Info.

Reply:
Done, changed to returning an empty string in both fields, which would be ignored under the current logic.